src/modules/ui/dom.js

```diff
@@ -252,6 +314,14 @@ BetterLyrics.DOM = {
     }
   },
 
+  /**
+   * Adds a button for users to contribute lyrics when none are found.
```

Comment:
```suggestion
   * Adds a button for users to contribute lyrics.
```
The code for the behavior happens at the caller